fleet/lib/rancher-backup/base/backup-s3-recurring.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: resources.cattle.io/v1
+kind: Backup
+metadata:
+  name: s3-recurring
+spec:
+  resourceSetName: rancher-resource-set
+  schedule: "@every 12h"
+  retentionCount: 10
```

Comment:
5 days seems too short.  That could run out during a holiday week.  I think we should have 30 days of retention as a minimum.  90+ days would be great as long as the size is reasonable.